pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -40,7 +40,7 @@ var replanThreshold = settings.RegisterFloatSetting(
 	"stream_replication.replan_flow_threshold",
 	"fraction of nodes in the producer or consumer job that would need to change to refresh the"+
 		" physical execution plan. If set to 0, the physical plan will not automatically refresh.",
-	0.1,
+	0,
```

Comment:
This seems unrelated.

Reply:
i'd like to turn the feature default off, as the patch skips the only e2e test that exercises this feature.